sql_connectors/connections.py

```diff
@@ -0,0 +1,33 @@
+def setup():
+    import os
+
+    def import_class(cl):
+        last_dot = cl.rfind(".")
+        class_name = cl[last_dot + 1 : len(cl)]
+        module_name = cl[0:last_dot]
+        try:
+            module = __import__(module_name, globals(), locals(), [class_name])
+        except ImportError:
+            raise Exception("Couldn't find module: {}".format(module_name))
+
+        try:
+            return getattr(module, class_name)
+        except AttributeError:
+            raise Exception("Couldn't find class: {}".format(class_name))
+
+    DEFAULT_PATH_OR_URI = os.environ.get(
+        "SQL_CONNECTORS_PATH_OR_URI", "~/.config/sql_connectors"
+    )
+
+    DEFAULT_STORAGE = os.environ.get(
+        "SQL_CONNECTORS_STORAGE", "sql_connectors.storage.LocalStorage"
+    )
+
+    default_backend = import_class(DEFAULT_STORAGE)
+    default_connections = default_backend(DEFAULT_PATH_OR_URI)
```

Comment:
should add some handling to this for cases where `default_backend` returns `None`

Reply:
that shouldn't happen since `import_class` would error out if it can't load anything. That being said, I moved this logic into a Traitlets based config as suggested by @wseaton 

See 3d3e301